api-metastore/src/main/java/org/zalando/nakadi/validation/schema/CompatibilityModeChangeConstraint.java

```diff
@@ -37,12 +38,14 @@ public CompatibilityModeChangeConstraint(final AdminService adminService,
         final boolean isNakadiAdmin = adminService.isAdmin(AuthorizationService.Operation.WRITE);
         final boolean isEventTypeAdmin = authorizationService
                 .isAuthorized(AuthorizationService.Operation.ADMIN, original.asResource());
-        final boolean isChangeValid = allowedChanges.get(original.getCompatibilityMode())
+        final boolean isChangeValid = original.getSchema().getType() == EventTypeSchemaBase.Type.AVRO_SCHEMA ||
```

Comment:
I do not understand the change here

Reply:
This would mean skip checking of compatibility mode type change when its a Avro type schema, I am not sure if it is desirable but it was done to reduce initial complexity for users.

Reply:
But again the complexity was supposed to be there when the new modes compatibility modes were added, since they are removed now I guess we could remove this change here?

Reply:
Reverted this to how it was originally, so this check will be performed whether its avro or json schema.